src/components/pages/manage/movies/create.tsx

```diff
@@ -1,67 +1,127 @@
 import { useMachine } from '@xstate/react';
-import type { MovieContext, MovieEvent } from 'machines/movie';
-import { movieMachine } from 'machines/movie';
```

Comment:
What is the reasoning that the state machine had to be moved into this file instead of being in a separate file?

Reply:
We're only using it on this page, so to me, it made a lot more sense to define it there.

Reply:
Okay was just wondering!